src/cascadia/TerminalSettingsModel/Profile.cpp

```diff
@@ -117,28 +102,19 @@ winrt::com_ptr<Profile> Profile::CopySettings() const
     profile->_Name = _Name;
     profile->_Source = _Source;
     profile->_Hidden = _Hidden;
-    profile->_Icon = _Icon;
-    profile->_CloseOnExit = _CloseOnExit;
-    profile->_TabTitle = _TabTitle;
     profile->_TabColor = _TabColor;
-    profile->_SuppressApplicationTitle = _SuppressApplicationTitle;
-    profile->_UseAcrylic = _UseAcrylic;
-    profile->_ScrollState = _ScrollState;
     profile->_Padding = _Padding;
-    profile->_Commandline = _Commandline;
-    profile->_StartingDirectory = _StartingDirectory;
-    profile->_AntialiasingMode = _AntialiasingMode;
     profile->_ForceFullRepaintRendering = _ForceFullRepaintRendering;

```

Comment:
i'm surprised you didn't move these into the macro . . . but it seems like you _did_?

Reply:
I'm sorry, what? This is supported as a profile setting?

Reply:
I'm very confused.

Reply:
They are in the global settings macro, not in the profile settings macro

Reply:
Because we don't actually use them in `Profile::LayerJson` or `Profile::ToJson`

Reply:
So... they just should have never been in `Profile` in the first place?

Reply:
@PankajBhojwani, can you follow up (separate PR) to remove these from Profile if they're not used? Thx